roles/setup_pxe_httpd/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Setup Httpd
+  block:
+    - name: Install Httpd
+      ansible.builtin.package:
+        name: httpd
+        state: present
+
+    - name: Configure httpd
+      ansible.builtin.template:
+        src: "./httpd.conf.j2"
+        dest: "/etc/httpd/conf.d/pxe_conf.conf"
+        owner: root
+        group: root
+        mode: 0644
+      notify: Restart httpd
```

Comment:
You already do this in `boot_iso`

Reply:
yes I moved that from there this is the draft file